spec/client_spec.js

```diff
@@ -148,7 +148,7 @@ describe('Client', () => {
       })
     })
 
-    it('throws when domain is invalid', () => {
+    it('clears iframe and appends error msg to body when domain is invalid', () => {
```

Comment:
Can we add one tesco domain to the test case above this?

Reply:
Done! should i add one to the invalid domains ?

Reply:
nah, that should be alright.